File: C++/282A-Bit++.cpp
/*
	Solution to Codeforces problem 282A
	Copyright (c) GeneralMing. All rights reserved.

	https://github.com/GeneralMing/codeforces
*/

#include<iostream>
#include<string>

using namespace std;

int main()
{
	int n, x = 0;
	bool flag;
	string inp;
	cin >> n;
	for(int i = 0; i < n; i++)
	{
		flag = false;
		cin >> inp;
		for(int j = 0; j < inp.length(); j++)
			if(inp.at(j) == '+')
				flag = true;
		if(flag)
			x += 1;
		else
			x -= 1;
	}
	cout << x << endl;
}

File: Java/282A-Bit++.java
/*
	Solution to Codeforces problem 282A
	Copyright (c) GeneralMing. All rights reserved.

	https://github.com/GeneralMing/codeforces
*/

import java.util.*;
public class Bit_282A
{
	public static void main(String[] args)
	{
		String inp;
		int n, x = 0;
		boolean flag;
		Scanner scanner = new Scanner(System.in);
		n = Integer.parseInt(scanner.nextLine());
		for(int i = 0; i < n; i++)
		{
			flag = false;
			inp = scanner.nextLine();
			for(int j = 0; j < inp.length(); j++)
				if(inp.charAt(j) == '+')
					flag = true;
			if(flag)
				x += 1;
			else
				x -= 1;
		}

		System.out.println(x);
	}
}